api/src/main/java/cdi/lite/extension/model/declarations/ClassInfo.java

```diff
@@ -0,0 +1,59 @@
+package cdi.lite.extension.model.declarations;
+
+import cdi.lite.extension.model.types.Type;
+import cdi.lite.extension.model.types.TypeVariable;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * @param <T> the inspected class
+ */
+public interface ClassInfo<T> extends DeclarationInfo {
+    String name();
+
+    String simpleName();
```

Comment:
```suggestion
    String getSimpleName();
```

Reply:
The same reason:
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Class.html#getSimpleName()